server/fileupdater.mjs

```diff
@@ -18,8 +18,13 @@ function updateProperties(properties, v) {
   if(typeof properties != 'object')
     return;
 
-  if(!properties.type)
-    updateProperties(properties.faces, v);
+  if(!properties.type) {
+    if (typeof properties.faces == 'object') {
+      for (let face in properties.faces) {
+        updateProperties(properties.faces[face], v);
+      }
+    }
+  }
```

Comment:
Hmm, crap, so this was completely broken since the beginning? :sob:

Reply:
Yup! Crafty exposed that this wasn't actually updating the face properties, heh.